Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [workspace]
 resolver = "2"
 members = [
+    "crates/anemo-cli",
```

Comment:
I think instead of making a new crate/binary we should probably bundle this with the `sui-tool` binary

Reply:
I thought about this and decided against it because this is more of an administrative/debugging tool whereas sui tool is more of a thing for end users. I'm not sure we really want to make sui tool even more of a kitchen sink

Reply:
AFAIK sui-tool is definitely a debugging tool and not intended for end users

Reply:
Ah sorry, I was definitely confusing sui-tool with the sui CLI. I do still think it makes sense for anemo CLI to be separate. The anemo tool is just a networking tool for direct interaction with that layer, without any smarts about how to actually craft valid request messages and so on (only the bare minimum of being able to serialize from a text format). It seems to me conceptually like a different use case than sui-tool (and also covers narwhal, not just sui). 

I don't have that strong of an opinion, so if you are insisting, I can do it. However the anemo-cli library is designed with the assumption that it will be the root of the clap binary and that you'll delegate your main function to it. I'd have to first go and refactor that to also support integrating with a subtree of existing clap binaries, if you think this is necessary.